patches/chromium/.patches

```diff
@@ -134,3 +134,4 @@ feat_enable_customizing_symbol_color_in_framecaptionbutton.patch
 cherry-pick-99cafbf4b4b9.patch
 cherry-pick-44b7fbf35b10.patch
 fix_potential_draggable_region_crash_when_no_mainframeimpl.patch
+cherry-pick-85dd0fcc70ed.patch
```

Comment:
Patches that modify dependencies have no effect as they're applied after the dependency is fetched.